CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+cmake_minimum_required (VERSION 3.13)
+
+project(SimpleParallelNoClock)
+
+add_definitions( -DLOGIC2 )
+
+# enable generation of compile_commands.json, helpful for IDEs to locate include files.
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+# custom CMake Modules are located in the cmake directory.
+set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
+
+include(ExternalAnalyzerSDK)
+
+set(SOURCES
+include/AnalyzerHelpers.h
+include/AnalyzerResults.h
+include/SimulationChannelDescriptor.h
+include/AnalyzerChannelData.h
+include/Analyzer.h
+include/LogicPublicTypes.h
+include/AnalyzerSettingInterface.h
+include/AnalyzerTypes.h
+include/AnalyzerSettings.h
+src/SimpleParallelNoClockAnalyzerResults.cpp
```

Comment:
To reduce the number of changes we're making to other people's code, I would suggest not modifying the folder names they are using for their source code, and instead just update this list here to have the right folder name.

Reply:
Step three here outlines renaming `source` to `src` https://github.com/saleae/SampleAnalyzer/#updating-an-existing-analyzer-to-use-cmake--github-actions. `include` was already in this repo so no renaming was done.